### PR TITLE
[ROSTESTS][SDK] Remove remnant 2KSP4 parts

### PR DESCRIPTION
**Pull request by SergeGautherie**
ReactOS does not target 2000 (SP4) anymore.

### CODE REVIEW

File: sdk/include/reactos/undocuser.h [hunk 1]
```diff
@@ -255,9 +255,7 @@ C_ASSERT(sizeof(MSGBOXDATA) == 0x88);
 
 #else
 
-#if (_WIN32_WINNT <= _WIN32_WINNT_WIN2K)    /* (NTDDI_VERSION <= NTDDI_WIN2KSP4) */
-C_ASSERT(sizeof(MSGBOXDATA) == 0x48);
-#elif (_WIN32_WINNT >= _WIN32_WINNT_WIN7)   /* (NTDDI_VERSION >= NTDDI_WIN7) */
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN7) /* (NTDDI_VERSION >= NTDDI_WIN7) */
```
Comment (HBelusca) [inline sdk/include/reactos/undocuser.h:258]: Please revert.

File: modules/rostests/win32/user32/messagebox/messagebox.c [hunk 2]
```diff
@@ -53,9 +53,7 @@ C_ASSERT(sizeof(MSGBOXDATA) == 0x88);
 
 #else
 
-#if (_WIN32_WINNT <= _WIN32_WINNT_WIN2K) /* (NTDDI_VERSION <= NTDDI_WIN2KSP4) */
-C_ASSERT(sizeof(MSGBOXDATA) == 0x48);
-#elif (_WIN32_WINNT >= _WIN32_WINNT_WIN7) /* (NTDDI_VERSION >= NTDDI_WIN7) */
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN7) /* (NTDDI_VERSION >= NTDDI_WIN7) */
```
Comment (HBelusca) [inline modules/rostests/win32/user32/messagebox/messagebox.c:56]: Please revert

**Review by HBelusca — CHANGES_REQUESTED** [review]

File: sdk/include/reactos/undocuser.h (hunk 1 above)
Comment (SergeGautherie) [reply sdk/include/reactos/undocuser.h:258]: Reverted, though I wonder why this code is still wanted. (Both files.)

Comment (HBelusca) [reply sdk/include/reactos/undocuser.h]: The person(s) who wrote that code, want to document what the size of this undocumented structure is on these different Windows versions.

**Review by HBelusca — APPROVED** [review]

File: sdk/include/reactos/undocuser.h (hunk 1 above)
Comment (SergeGautherie) [reply sdk/include/reactos/undocuser.h]: It seems you were the author, 3 years ago. Even at that time, ReactOS (build) did not care about 2000 anymore, did it?
If not removing this "documentation", I would suggest to switch to some kind of comment. Anyway...

**Pull request merged**